KEYB.INC

```diff
@@ -0,0 +1,15 @@
+;-------------------------------------------------------------------
+; 32-bit Keyboard functionality.
+;-------------------------------------------------------------------
+
+GLOBAL __keyb_install_keyboard_handler:PROC
+GLOBAL __keyb_uninstall_keyboard_handler:PROC
+
+GLOBAL __keyb_keyboard_state:BYTE:128
```

Comment:
the keyboard_state, raw_scanc_code and keys_active etc shouldn't be global. On top of that, since they will be local to the KEYB.ASM file drop the leading '__'